zio-http/shared/src/main/scala/zio/http/endpoint/openapi/JsonSchema.scala

```diff
@@ -254,111 +254,138 @@ object JsonSchema {
       .get
 
   private[openapi] def fromSerializableSchema(schema: SerializableJsonSchema): JsonSchema = {
-    val additionalProperties = schema.additionalProperties match {
-      case Some(BoolOrSchema.BooleanWrapper(bool))  => Left(bool)
-      case Some(BoolOrSchema.SchemaWrapper(schema)) =>
-        val valuesSchema = fromSerializableSchema(schema)
-        Right(
-          schema.optionalKeySchema.fold(valuesSchema)(keySchema =>
-            valuesSchema.annotate(
-              MetaData.KeySchema(
-                fromSerializableSchema(keySchema),
+
+    val definedAttributesCount = schema.productIterator.count(_.asInstanceOf[Option[_]].isDefined)
```

Comment:
One thing I learned from the zio http open api integration: I hate it and open API is a crap format haha 😂